app/components/calls_to_action/get_support_block_component.rb

```diff
@@ -0,0 +1,26 @@
+module CallsToAction
+  class GetSupportBlockComponent < ViewComponent::Base
+    attr_reader :title, :image, :title_color, :links
+
+    include ContentHelper
+
+    def initialize(
+      title: "Get free support",
+      image: "get-support-block-promo.jpg",
+      title_color: "yellow",
+      links: [
+        { text: "Get an adviser", url: "/teacher-training-adviser/sign_up/identity" },
```

Comment:
Should this go to the TTA page rather than the sign-up form? @emmaachesongray to advise